CONTRIBUTING.md

```diff
@@ -61,14 +61,7 @@ cargo install cargo-insta
 And you'll need pre-commit to run some validation checks:
 
 ```shell
-pipx install pre-commit  # or `pip install pre-commit` if you have a virtualenv
-```
-
-You can optionally install pre-commit hooks to automatically run the validation checks
-when making a commit:
-
-```shell
-pre-commit install
```

Comment:
Why remove all of this?

Reply:
I see it moved down, I'm not sure that's clearer.

Reply:
Sorry I didn't have the chance to add comments yet.

This was because during setup, users are doing each thing step by step.

But at this point, the user isn't up to the step where the repo is downloaded, to be able to setup the pre-commit hooks in.